### PR TITLE
Cleanup some code for translation

### PR DESCRIPTION
**Pull request by FransMeerhoff**
- Fixed typos
- Remove empty strings 
- Unwanted spaces
- remove messages for developers
- EOL in gettext is not wanted
- ifthey -> if they

### CODE REVIEW

File: JASP-Engine/JASP/R/LScommon.R [hunk 1]
```diff
@@ -114,8 +114,7 @@ hdi.density    <- function(object, credMass=0.95, allowSplit=FALSE, ...) {
   gaps <- which(diff(indices) > 1)
   if(length(gaps) > 0 && !allowSplit) {
     # In this case, return shortest 95% CrI
-    warning("The HDI is discontinuous but allowSplit = FALSE;
```
Comment (vandenman) [inline JASP-Engine/JASP/R/LScommon.R:117]: @Kucharssim is it fine to remove this warning, or do you catch it somewhere to show as a footnote?

**Review by vandenman — APPROVED** [review]
The changes look fine to me, but I can't judge whether it's ok to remove the warning.

**Comment by FransMeerhoff** [comment]
@vandenman I discussed that with František about the warning, should be Ok

File: JASP-Engine/JASP/R/LScommon.R (hunk 1 above)
Comment (Kucharssim) [reply JASP-Engine/JASP/R/LScommon.R:117]: Sorry, I did not see the mention - this is not my file! 😄

**Pull request merged**